src/drivers/sensors/dht.c

```diff
@@ -210,7 +202,111 @@ static int read_sensor(struct dht_driver_t *self_p, uint8_t *buf_p)
         return (-EPROTO);
     }
 
-    return 0;
+    return (0);
+}
+
+#if CONFIG_DHT_COMMAND_READ == 1
+
+static int cmd_read_sensor(int argc,
+                           const char *argv[],
+                           void *chout_p,
+                           int (*read_p)(struct dht_driver_t *, float *, float *),
+                           const char *fn_name_p)
+{
+    struct dht_driver_t dht;
+    int pin;
+    int res;
+    float temperature;
+    float humidity;
+
+    if (argc != 2) {
+        std_printf(OSTR("Usage: %s <pin name>\r\n"), fn_name_p);
+
+        return (-EINVAL);
+    }
+
+    pin = board_pin_string_to_device_index(argv[1]);
+
+    if (pin < 0) {
+        std_printf(OSTR("Invalid pin name.\r\n"));
```

Comment:
Add the pin name (`args[1]`) to the string to help the user understand which argument was wrong.